python_modules/dagster/dagster/_core/storage/alembic/versions/047_add_run_tags_run_id_idx.py

```diff
@@ -0,0 +1,54 @@
+"""add run_id to run_tags index
+
+Revision ID: 6b7fb194ff9c
+Revises: 16e3655b4d9b
+Create Date: 2024-11-14 22:19:46.456731
+
+"""
+
+from alembic import op
+from dagster._core.storage.migration.utils import has_index, has_table
+
+# revision identifiers, used by Alembic.
+revision = "6b7fb194ff9c"
+down_revision = "16e3655b4d9b"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    if has_table("run_tags"):
+        if not has_index("run_tags", "idx_run_tags_run_id"):
+            op.create_index(
+                "idx_run_tags_run_id",
+                "run_tags",
+                ["key", "value", "run_id"],
+                unique=True,
+                postgresql_concurrently=True,
+                mysql_length={"key": 255, "value": 255, "run_id": 255},
```

Comment:
Without these values, the mysql backcompat migration test was failing.
I tried to look for the values in the alembic migration history, but was unsuccessful. 
Please validate.

Reply:
The existing schema has `key`/`value` at 64, and `run_id` at 255.

Reply:
Thanks, fixed.